certbot-apache/certbot_apache/configurator.py

```diff
@@ -511,7 +511,11 @@ def _create_vhost(self, path):
 
         """
         addrs = set()
-        args = self.aug.match(path + "/arg")
+        try:
+            args = self.aug.match(path + "/arg")
+        except RuntimeError:
```

Comment:
I think this could be triggered in various different occasions, so maybe more generic error message would be in order. Also it'd be best to print out the path that caused the error for the user to notify exactly what caused the breakage.